### ISSUE TITLE
temporal lobe atrophy

### ISSUE THREAD

**Issue by sarahstofel** [mgijax/mammalian-phenotype-ontology#3423]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

temporal lobe atrophy

>2. Synonyms (e.g., absent spleen)

atrophy of temporal lobe

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

acquired atrophy of the lower lateral part of the cerebral hemisphere

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

https://www.sciencedirect.com/science/article/pii/S0091305796004200?via%3Dihub

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

forebrain atrophy

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

https://orcid.org/0000-0003-2307-1226

**Comment by sbello** [comment]
MP:0021008

**Comment by anna-anagnostop** [comment]
I think the synonym "atrophy of temporal lobe" should be a **broad** rather than an exact synonym of temporal cortex atrophy MP:0021008.

See temporal lobe [UBERON:0001871](http://purl.obolibrary.org/obo/UBERON_0001871) and temporal cortex [UBERON:0016538](http://purl.obolibrary.org/obo/UBERON_0016538). Per UBERON, temporal cortex is defined as: "Gray matter of the temporal region of the neocortex, located in the temporal lobe in gyrencephalic animals"

The EQ for MP:0021008 currently uses "temporal lobe", rather than "temporal cortex" to reflect the given textual definition. Perhaps we need both terms: temporal lobe atrophy and temporal cortex atrophy.

**Comment by sbello** [comment]
Changed term label to lobe. The issue hereis that lobe and cortex are used differently in rodent vs human. We need to revisit the labels used in MP eventually.
